services/iam/drivers/IamAccount.py

```diff
@@ -238,8 +239,15 @@ def _checkHasOrganization(self):
                 return 0
     
     def _checkCURReport(self):
-        results = self.curClient.describe_report_definitions()
-        if len(results.get('ReportDefinitions')) == 0:
-            self.results['enableCURReport'] = [-1, '']
+        try:
+            results = self.curClient.describe_report_definitions()
+            if len(results.get('ReportDefinitions')) == 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)